### PR TITLE
Update README.md for project

### PR DESCRIPTION
**Pull request by nerrad**
This expands the readme.md to serve as a resource giving an overview of all the related components of the ongoing php related initiatives tied to "servehappy".  Intended to be a single resource that can be linked to (already linked to via the #core-php slack channel topic) for helping people get up to speed on what the "servehappy" project is all about and where to find all related initiatives.

Changes/improvements include:

- clearly articulating some goals
- giving some background and where this fits into the overall php related initiatives currently in process within the WordPress project.
- providing some general overview of all ongoing php language initiatives in the WordPress project (so this can be used as a resource for connecting them all together).
- providing links to all the different in process work.
- providing links to various communication channels.

I think this dovetails nicely with the work @felixarntz has been doing in #36    (which can be linked to from this README.md as well).

Is this useful/helpful?  Any changes/improvements to make?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,85 @@
...
+The servehappy project is an initiative that seeks to educate WordPress site owners about the value of having the latest version of PHP powering their WordPress site.  This supports the overall goal of increasing percentage of WordPress installs running modern PHP versions.
 
-> Note: `servehappy` is only a temporary codename for now.
+## Background
 
-This is the main repository for working on the `servehappy` page(s). It is used to collect the discussions in the issue tracker and work on the page content.
+One of the core tenets of WordPress has been that of being backward compatible and the support of PHP 5.2 is and has been part and parcel of that tenet.  Over the years however, there has been increasing tension within the WordPress developer community on this support for PHP5.2 between those who support keeping the minimum PHP version requirements the same for WordPress core, and those who want to increase the minimum PHP version requirements for WordPress core.  There have been numerous arguments made for either side and you can follow some of the discussion [here](https://core.trac.wordpress.org/ticket/33381).
```
Comment (grappler) [inline README.md]: I think there should be a space between `PHP` and `5.2` in the second time in the paragraph.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,85 @@
...
 
-* [Current Draft](https://github.com/WordPress/servehappy/blob/master/DRAFT.md)
+Beginning around mid 2017, there were a number of discussions initiated (in various channels) about resolving some of this tension and keeping WordPress as a project moving forward with PHP language related topics. Out of these discussions, a number of initiatives were started that support the overall _initial_  goals of:
 
-## Collecting Benefits & Stats
+- encouraging Web host providers to provide the latest PHP versions and provide an upgrade path for their customers to get on the latest versions (this is something that has been happening for a while now)
+- educating site owners about PHP and why PHP version matters.
+- decreasing the amount of reported WordPress sites running older EOL versions of PHP.
```
Comment (grappler) [inline README.md]: It would be good to start the sentences with capitals and perhaps explain what EOL stands for.

File: README.md (hunk 1 above)
Comment (nerrad) [reply README.md]: Done in latest commit

File: README.md (hunk 2 above)
Comment (nerrad) [reply README.md]: Done in latest commit.

Comment (nerrad) [reply README.md]: For EOL I added a brief note about it `(End of Life)` and linked to the Wikipedia entry on End of Life.  I also recognized an opportunity to link the 'PHP' text in this to the supported versions document on php.net.

**Pull request merged**